frontend/src/pages/faq.page.tsx

```diff
@@ -10,308 +10,294 @@ const Faq: NextPage = () => {
 
   return (
     <Layout theme="free">
-      <main>
-        <div className={styles["faq-page"]}>
```

Comment:
This looks like a very large diff, but literally the only thing I did was remove this `<div>` and move the class `.faq-page` to `<main>` - but of course, that changed the indentation.

The reason I did this was because the `<div>` was superfluous, and while it had a `height: 100%` set on it (to make sure the footer sticks to the bottom), the containing `<main>` did not have that. Since a percentage height is relative to the container, this (I think) caused the height of the `<div>` to change a little bit during page load on Firefox, causing things to jump around and thereby the screenshot to be of a different height depending on how soon it was taken. (Resulting in a flaky test.) Since it didn't do anything, I just removed it.